build.sh

```diff
@@ -107,6 +107,13 @@ install_ocp_tools() {
         && mv oc /usr/bin
 }
 
+install_rhel_gpg_key() {
+    if [ ! -f /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release ]; then
+        curl -Lo /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release \
```

Comment:
Note that in theory we should be verifying the key fingerprint here.
But we already ship `distribution-gpg-keys` in cosa, why don't we symlink this instead?

Reply:
Ahh nice, hadn't thought to check there. Will test and update.

Reply:
Updated!

I think though we probably want a cleaner long-term solution for this so that we use the strictest keyring necessary when building each of FCOS, RHCOS and SCOS.